### DISCUSSION TITLE
Heavy or light chain of BCR CDR3

### DISCUSSION THREAD

**Discussion by seongguheo** [milaboratory/mixcr#1244] [Q&A] [answered]
Dear Authors,

Hello, 
I am looking at  Smart-seq2 full length scRNA-seq data for BCR CDR3 clonal sequence.
I wonder which one, heavy or light chain, does mixcr use to predict CDR3 sequence.
Is there any preference to chose the chain?
Please, have a look the below, some  of the clones.tsv file of my analysis.

-----------<from of the clones.tsv file>-------------------------------------------------------
aaSeqCDR3  allVHitsWithScore allDHitsWithScore               allJHitsWithScore allCHitsWithScore
1    CQQSYSLPLTF IGKV1-39*00(392.2)                                   IGKJ4*00(188.1)    IGKC*00(230.3)
2   CQSYDTSLSGVF IGLV1-40*00(461.1)                                   IGLJ3*00(213.4)   IGLC3*00(215.9)
3   CASYTITLTYVF IGLV2-14*00(385.8)                                   IGLJ1*00(184.8)   IGLC1*00(198.7)
4    CQQYNTYPWTF  IGKV1-5*00(290.7)                                   IGKJ1*00(189.5)      IGKC*00(181)
5 CQVWETSGDPYVVF IGLV3-21*00(566.1)                                   IGLJ2*00(176.1)   IGLC2*00(173.8)
6   CSSYTNSRSWIF IGLV2-14*00(682.4)                   IGLJ3*00(165.5),IGLJ2*00(164.6)   IGLC2*00(176.4)
----------------------------------------------------------------------------------------------------------

Many thanks,
Luke

**Comment by mizraelson** [comment, marked as answer]
Hi,
The choice between heavy and light chain depends on the covered V and J genes. mixcr uses all chains  references (for the selected species) to align the reads and then assembles the clones accordingly.
